src/lib/Libattr/master_resv_attr_def.xml

```diff
@@ -852,6 +852,23 @@
       </member_at_type>
       <member_at_parent>PARENT_TYPE_RESV</member_at_parent>
    </attributes>
+   <attributes>
+      <member_index>RESV_ATR_start_revert</member_index>
+      <member_name>ATTR_resv_start_revert</member_name>
+      <member_at_decode>decode_l</member_at_decode>
+      <member_at_encode>encode_l</member_at_encode>
+      <member_at_set>set_l</member_at_set>
+      <member_at_comp>comp_l</member_at_comp>
+      <member_at_free>free_null</member_at_free>
+      <member_at_action>NULL_FUNC</member_at_action>
+      <member_at_flags>READ_WRITE </member_at_flags>
```

Comment:
make this READ_ONLY, and possibly only by managers

Reply:
How about if I make this PRIV_READ?

Reply:
ATR_DFLAG_MGRD is probably better because this is an internal value that only the scheduler/pbs_sim/pbs_est  need.  You need to make managers see it because pbs_est runs as a root which is a manager.